physionet-django/templates/home.html

```diff
@@ -33,39 +33,22 @@
       <div class="col-md-7">
         <h1>New Content</h1>
         <hr>
-        {# A publication #}
-        {% for published_project, authors , topics in projects_authors_topics %}
-          <p>{{ published_project.resource_type|resource_badge|safe }}
-             {{ published_project.access_policy|access_badge|safe }}
-          </p>
-          <a href="{% url 'published_project' published_project.slug %}"><h2>{{ published_project.title }}</h2></a>
-          {% if not published_project.is_legacy %}
-            <p class="text-muted">
-              <strong>Authors:
-              {% for author in authors %}
-                {{ author.get_full_name }}
-              {% endfor %}
-              </strong>
-            </p>
-          {% endif %}
-          {{ published_project.abstract|safe }}
-          <p class="text-muted">
-            Published: {{ published_project.publish_datetime|date }}.
-            Version: {{ published_project.version }}
-            <br>
-            {% for topic in topics %}
-              <a href="{% url 'topic_search' topic.description %}"><span class="badge badge-info">{{ topic }}</span></a>
-            {% endfor %}
-          </p>
-          <hr>
-        {% endfor %}
-
-        <a class="btn btn-outline-primary" href="{% url 'content' %}">More Content</a>
+        {% include "search/content_list.html" %}
+        <a class="btn btn-outline-dark" href="{% url 'content_index' %}">More Content</a>
         <br><br>
       </div>
 
       <!--Right Column -->
       <div class="col-md-5">
+
+        <div class="card">
+          <h2 class="card-header">Welcome to PhysioNet</h2>
+          <div class="card-body">
+            <p class="card-text mt-1">PhysioNet is a massive digital library and publisher of physiologic <a href="{% url 'database_index' %}">data</a> and <a href="{% url 'software_index' %}">software</a>, helping to make biomedical research more transparent and reproducible.</p>
```

Comment:
Perhaps stick with language from the current site. e.g. "PhysioNet offers free web access to large collections of recorded physiologic signals and related open-source software. For an overview of the content, see our XXX page."

Reply:
Signals? Web access? Are you sure we want to keep exactly as is?

Reply:
My general view is the more the new site looks like the current site, the easier it will be to make the move. We can tweak text later..